src/entity/light.cpp

```diff
@@ -0,0 +1,26 @@
+#include <light.h>
+
+Light::Light() : location(Point()), intensity(Intensity()) {}
+
```

Comment:
Acho que a gente não deveria permitir isso. Mas caso queira permitir:

```suggestion

Light::Light() {}

```
os outros já vão por padrão